blog/2023-07-23-openhab-4-0-release.md

```diff
@@ -0,0 +1,244 @@
+
+In addition to the icons already provided via the openHAB server (classic icon set + custom user icons), you have now easy access for your items and sitemap elements to the [Material icons](https://fonts.google.com/icons?icon.set=Material+Icons), [Framework7 icons](https://framework7.io/icons/) and [over 150,000 icons provided by Iconify](https://icon-sets.iconify.design/).
+
+Main UI supports all these icon sources while Basic UI currently supports all of them except the Framework7.
+Note that the Iconify option requires internet connectivity on the client to access the external API.
+The web browser will cache the retrieved icons to limit the requests and speed up the rendering.
+Basic UI provides a setting to enable this option which is disabled by default.
+
+Dynamic icons based on the item state remains a feature only available with the openHAB icon sets.
+
+For more details including the new category/icon syntax, please read [the documentation](/docs/configuration/items.html#icons).
+
+#### Managing your icon color
+
+You can now control the icon color associated with any user interface element of a sitemap using the "iconcolor" parameter. The syntax and capabilities behind this parameter are the same as for the "labelcolor" and "valuecolor" parameters, which means you can define multiple conditions leading to different colors.
+
+This is not possible with icons from the classic icon set because these icons have a hard-coded color palette, but it is possible with icons from other icon sources and with added user SVG icons containing "currentColor" as the fill color.
+Basic UI supports this color feature for openHAB icons only if an option is enabled.
+Color support for other icon sources is supported by default.
+
+For more details, please read [the documentation](/docs/ui/sitemaps.html#label-value-and-icon-colors).
+
+## New input field for sitemap UIs
+
+No new user interface elements have been added for sitemap UIs since openHAB version 1. It finally arrives in version 4 with the introduction of a new UI element expected by users, namely the input field.
+
+```java
+Input item=<itemname> [label="<labelname>"] [icon="<iconname>"] [inputHint="<inputHint>"]
+```
+
+This element allows entering of text, numbers and dates/times and updating the underlying items.
+Text, number or number with unit values and dates/times can be updated in the respective item types.
+
+The parameter `inputHint` will give a hint to the user interface to use a specific widget adapted to a specific use. For example, it can be a time picker or a date picker.  Valid values for the parameter are `text`, `number`, `date`, `time` and `datetime`. Support of these widget types vary by user interface and/or browser.
+
+This new input field is already supported in Basic UI and in the Android app.
+
+Many thanks to Mark Herwege who fully implemented this new For examplefeature (sitemap syntax update in core framework, implement in Basic UI and in the Android app).
```

Comment:
```suggestion
Many thanks to Mark Herwege who fully implemented this new feature (sitemap syntax update in core framework, implement in Basic UI and in the Android app).
```

Reply:
Done in https://github.com/openhab/website/pull/412/commits/db992015e75b37451e0eed0381e7eee8827b68c9.